.cirrus.yml

```diff
@@ -15,16 +15,22 @@ task:
       type: text/json
       format: golangci
 
-task:
-  name: Test (Linux)
+docker_builder:
+  name: Test (Linux with Docker)
```

Comment:
Let's rename it back since it breaks required checks

Reply:
I'm not sure why, because it uses aliases, but renamed it in https://github.com/cirruslabs/cirrus-ci-agent/pull/253/commits/654ff83699fca2603fa04a1c30c156439cbed514.